core/src/test/java/org/bitcoinj/base/LegacyAddressTest.java

```diff
@@ -139,29 +129,6 @@ public void getNetwork() {
         assertEquals(TESTNET, testNet);
     }
 
-    @Test
```

Comment:
If we remove this test, shouldn't we better remove the entire cap to register other networks? (We can still deprecate these methods in 0.17 last minute.)

Reply:
I'm not sure where else in `core` relies on this list. This PR is just to remove use of registering Network from address parsing and related functionality in base.

Reply:
> I'm not sure where else in core relies on this list. 

For example `BitcoinURITest.testConvertToBitcoinURI()` as you point out. There may be other places as well. We should remove these separately and probably on a case-by-case basis.